modules/coreI18n/src/main/key.scala

```diff
@@ -2630,6 +2632,35 @@ object I18nKey:
     val `doubleDefeat`: I18nKey = "study:doubleDefeat"
     val `blackDefeatWhiteCanNotWin`: I18nKey = "study:blackDefeatWhiteCanNotWin"
     val `whiteDefeatBlackCanNotWin`: I18nKey = "study:whiteDefeatBlackCanNotWin"
+    val `localAnalysis`: I18nKey = "study:localAnalysis"
+    val `serverAnalysis`: I18nKey = "study:serverAnalysis"
+    val `currentGame`: I18nKey = "study:currentGame"
+    val `cannotBePublished`: I18nKey = "study:cannotBePublished"
+    val `projected`: I18nKey = "study:projected"
+    val `xNodesPerMove`: I18nKey = "study:xNodesPerMove"
+    val `has`: I18nKey = "study:has"
+    val `willUse`: I18nKey = "study:willUse"
+    val `quality`: I18nKey = "study:quality"
+    val `standardQuality`: I18nKey = "study:standardQuality"
+    val `broadcast`: I18nKey = "study:broadcast"
+    val `broadcastQuality`: I18nKey = "study:broadcastQuality"
+    val `ultra`: I18nKey = "study:ultra"
+    val `ultraQualityXSeconds`: I18nKey = "study:ultraQualityXSeconds"
+    val `doneInX`: I18nKey = "study:doneInX"
+    val `nodes`: I18nKey = "study:nodes"
+    val `moveXOfY`: I18nKey = "study:moveXOfY"
+    val `xTimesYQuality`: I18nKey = "study:xTimesYQuality"
+    val `publish`: I18nKey = "study:publish"
+    val `analysisUploadFailed`: I18nKey = "study:analysisUploadFailed"
+    val `serverAnalysisInProgress`: I18nKey = "study:serverAnalysisInProgress"
+    val `youCanPublish`: I18nKey = "study:youCanPublish"
+    val `turnOnRec`: I18nKey = "study:turnOnRec"
+    val `whenUpgradingOldChapters`: I18nKey = "study:whenUpgradingOldChapters"
+    val `looksLikeASimilar`: I18nKey = "study:looksLikeASimilar"
+    val `keepMine`: I18nKey = "study:keepMine"
+    val `useTheirs`: I18nKey = "study:useTheirs"
+    val `serverAlreadyHas`: I18nKey = "study:serverAlreadyHas"
+    val `startingPosition`: I18nKey = "study:startingPosition"
```

Comment:
I usually prefer to dev new features without translations, then make them translatable as they're finalized, to reduce translator churn while the feature is being worked on, but ok.

This looks like enough new keys to be moved to a new file. They're also not study specific

Reply:
> They're also not study specific

ah but there's no analyse.xml.

> This looks like enough new keys to be moved to a new file.

we'd need a new on-demand i18n fetch mechanism (i'd grab it from the loadEsm initModule) unless you want to deliver these translations on every analysis page?

it would need lila to resolve the i18n hash because we don't want to deliver the i18n manifest to the client, it's huge. but the lila i18n resolver endpoint could be useful in other situations so i'm not against it at all.

Reply:
then again, realistically, it's not the library of congress. probably under 8k

Reply:
it's a separate PR though right?